Doc/library/datetime.rst

```diff
@@ -1167,7 +1166,7 @@ Instance attributes (read-only):
    In ``[0, 1]``. Used to disambiguate wall times during a repeated interval. (A
    repeated interval occurs when clocks are rolled back at the end of daylight saving
    time or when the UTC offset for the current zone is decreased for political reasons.)
-   The value 0 (1) represents the earlier (later) of the two moments with the same wall
+   The value ``0`` (1) represents the earlier (later) of the two moments with the same wall
```

Comment:
Why is `0` formatted, but `1` is not?

Reply:
Is it a reference to L1195? There is a similar reference later in the docs. `0` is marked up since it is a literal.

Reply:
Reading again, I think the `1` is a number and not a reference... `0` is earlier and `1` is later

Reply:
Correct. See also PEP-495.